yarn.lock

```diff
@@ -3015,14 +3015,14 @@
   dependencies:
     "@types/node" "*"
 
-"@types/node@*", "@types/node@>=10.0.0", "@types/node@>=8.1.0", "@types/node@^20.8.0":
+"@types/node@*", "@types/node@>=10.0.0", "@types/node@>=8.1.0":
```

Comment:
Kan je verklaren waarom yarn.lock is gewijzigd, zou normaal niet mogen als je geen wijzigingen hebt gemaakt in de package.json